inc/tag.class.php

```diff
@@ -440,6 +442,18 @@ static function preItemForm($params = []) {
       }
    }
 
+   /**
+    * Display the current tag dropdown in form header of items
+    *
+    * @param  array $params should contains theses keys:
+    *                          - item the CommonDBTM object
+    * @return nothing
+    * @deprecated 2.8.0
+    */
+   static function preItemForm($params = []) {
+      Toolbox::deprecated('Use PluginTagTag::showForItem instead');
+   }
+
```

Comment:
There is no need to handle deprecation on plugins classes, unless you know another plugin that calls this method.
```suggestion
```